site/source/index.blade.php

```diff
@@ -1,5 +1,11 @@
 @extends('_layouts.master')
 
+{{-- Meta Data --}}
+@if ( $page->title = 'Page title' )@endif
+@if ( $page->description= 'Description' )@endif
+@if ( $page->keywords= 'Keywords, best' )@endif
+
+{{-- Content --}}
```

Comment:
We can use YAML front matter for adding variables to Blade templates, so this can be simplified to:
```
---
title: Page Title
description: Page Description
keywords: Page, Keywords
---

@extends('_layouts.master')
...
```